### ISSUE TITLE
Cannot reproduce results from notebook on Colab

### ISSUE THREAD

**Issue by jespernwulff** [keras-team/keras-hub#1592]
**Describe the bug**
Running the [KerasNLP guide](https://keras.io/keras_nlp/) in Colab, I get more or less 50% classification accuracy, i.e. no better than a random classifier. It get the same when fine-tuning the classifier. I am making no changes to any code in the notebook. Just running the code. 

**To Reproduce**
Link to the [Colab Notebook](https://colab.research.google.com/github/keras-team/keras-io/blob/master/guides/ipynb/keras_nlp/getting_started.ipynb), which reproduces the bug.

**Expected behavior**
I expected to get the same accuracy results as reported in the text, i.e., around 78% accuracy for the first example.

**Comment by SuryanarayanaY** [comment]
Hi @jespernwulff ,

I have replicated the issue and attached [gist](https://colab.sandbox.google.com/gist/SuryanarayanaY/c3f9d40789716a698f6f449a2ece66cd/1592.ipynb) here. 

In the generated documentation [guide](https://keras.io/guides/keras_nlp/getting_started/) the `sparse_categorical_accuracy` touced to `0.8443` in 2nd epoch but when trained now it is staying at `0.5` only even in epoch 3.

**Comment by SamanehSaadat** [comment]
Hi @jespernwulff!

Thanks for reporting this issue! I'll look into it!

**Comment by SamanehSaadat** [comment]
I think the issue is related to the dataset. I also got around 50% accuracy when I ran the fine-tuning with the IMDB dataset downloaded in the guide but when I switched to loading the IMDB dataset from `tensorflow_datasets`, I got 87% accuracy.

I'll update the guide to get the IMDB dataset from `tensorflow_datasets` for now.

**Comment by dhantule** [comment]
Hi @jespernwulff, thanks for reporting this. 

KerasNLP is now renamed to KerasHub, please refer this updated [guide](https://keras.io/keras_hub/getting_started/), let me know if you face any issues. Thanks!
